src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
The interface name 'Draw' is a verb, which is not recommended for class or interface names. Consider using a noun or a noun phrase that describes the capability, such as 'Drawable' or 'ShapeRenderer', to improve the readability and intent of the interface.